ui/admin/package.json

```diff
@@ -34,6 +34,10 @@
     "test": "concurrently \"npm:test:*\" --names \"test:\"",
     "test:ember": "COVERAGE=true ember exam --test-port 0 --split=4 --parallel=1 --random",
     "e2e": "yarn playwright test --config ./tests/e2e/playwright.config.js",
+    "e2e:ce:aws": "yarn run e2e --grep \"(?=.*@ce)(?=.*@aws)\"",
```

Comment:
what is the purpose of `-grep \"(?=.*@ce)(?=.*@aws)\"` ?

Reply:
That's currently the only way to do AND with tags (run tests with tag `ce` AND tag `aws`)

From: https://playwright.dev/docs/test-annotations#tag-tests
> Or run tests containing both tags (logical AND operator) using regex lookaheads:
> `npx playwright test --grep "(?=.*@fast)(?=.*@slow)"`